.circleci/ansible/bin/install_dependencies.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# TODO: Eventually this step will not be required since these packages will be part of the custom image (which hasn't been created yet).
+# TODO: See "Codify the ova build Infrastructure #8": https://github.com/StackStorm/ova/issues/8
+###### PROVISION SERVER
+
+# Install packer
+PACKER_VERSION='1.2.3'
```

Comment:
Packer is installed via `makefile`.

let's please use that for a single source of truth.